### ISSUE TITLE
Suddenly error when i tried to run

### ISSUE THREAD

**Issue by kurroo10** [highcharts/highcharts#21706]
#### Expected behaviour
Application Running Without error

#### Actual behaviour
```js
Module parse failed: Unexpected token (6211:16)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
 * ./node_modules/source-map-loader/dist/cjs.js
You may need an additional loader to handle the result of these loaders.
|             item: t
|           }));
>         }, this), d(i, e.plotHeight)) t = o.item.legendItem || {}, y(o.pos) && (t.y = e.plotTop - e.spacing[0] + o.pos);
|       }
|       render() {

```
#### Product version

"highcharts": "11.2.0",
"highcharts-react-official": "3.2.1",

#### Affected browser(s)
Edge, Chrome, Firefox, Safari

**Comment by eh1160** [comment]
We have a similar build error just started happening in many of our Angular repos:
```
./node_modules/highcharts/highcharts.js:6193:16 - Error: Module parse failed: Unexpected token (6193:16)
File was processed with these loaders:
 * ./node_modules/@angular-devkit/build-angular/src/tools/babel/webpack-loader.js
You may need an additional loader to handle the result of these loaders.
|             item: t
|           }));
>         }, this), d(i, e.plotHeight)) t = o.item.legendItem || {}, y(o.pos) && (t.y = e.plotTop - e.spacing[0] + o.pos);
|       }
|       render() {
```
"highcharts": "11.3.0",
devDependency: "@babel/core": "7.24.0",

**Comment by SebastianBock83** [comment]
Similar problem here when using regular **graphs**:
We are currently using `11.4.6` or ´11.4.7` of highcharts.

`
[webpack-dev-server] ERROR in ../node_modules/highcharts/highcharts.js 422:26
Module parse failed: Unexpected token (422:26)
File was processed with these loaders:
 * ../node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
|       replaceNested: function (t, ...e) {
|         let i, s;
>         do for (s of i = t, e) t = t.replace(s[0], s[1]); while (t !== i);
|         return t;
|       },
`

**heatmaps**:
`
[webpack-dev-server] ERROR in ../node_modules/highcharts/modules/heatmap.js 803:43
Module parse failed: Unexpected token (803:43)
File was processed with these loaders:
 * ../node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
|           this.processedXData = t, this.processedYData = e;
|         }
>         for (let e of this.generatePoints(), this.points) {
|           let r = e.getCellAttributes(),
|             l = Math.min(r.x1, r.x2),
`

Best Regards

**Comment by J4ndr3** [comment]
Same problem any workarounds in the meantime?

**Comment by pascalmann** [comment]
Fixed by https://github.com/babel/babel/pull/16764.

As long as `@angular-devkit/build-angular` dependency is not updated, add this in your package.json (yarn):
```
"resolutions": {
	"@babel/generator": "^7.25.5"
},
```

**Comment by J4ndr3** [comment]
After the babel release earlier the problem is fixed on my side.

**Comment by horizoncarlo** [comment]
Same issue here, going to downgrade Highcharts for now and see if that helps

**Comment by kurroo10** [comment]
> Fixed by [babel/babel#16764](https://github.com/babel/babel/pull/16764).
> 
> As long as `@angular-devkit/build-angular` dependency is not updated, add this in your package.json (yarn):
> 
> ```
> "resolutions": {
> 	"@babel/generator": "^7.25.5"
> },
> ```

how about on react CRA ?
